merge_command.go

```diff
@@ -180,3 +202,24 @@ func handleMergeConflict(issue Issue, issues Issues) *ErrorResponse {
 	}
 	return nil
 }
+
+func removeRemoteBranch(pr *github.PullRequest, gitRepos git.Repos) *ErrorResponse {
```

Comment:
No reason really https://github.com/salemove/github-review-helper/pull/28/commits/08f1cb3c633e2c50f3b9ad66a42b31957a089174. Initially the `git` package method also has a _remove_ prefix, but I changed because of git's `--delete`.